plasoscaffolder/helpers/git.py

```diff
@@ -92,13 +100,18 @@ def SwitchToBranch(self, branch: str) -> int:
     return exit_code
 
   def CreateFeatureBranch(self, branch: str = '', module_name: str = ''):
-    """Creates a feature branch in the git project.
+    """Creates a feature branch in the git project if not exists.
```

Comment:
This is still confusing - it's going to create a branch, but it's also not going to create it, per line 111? 

Also, this is never called with a branch argument that I can see, so why not just remove it to make this code clearer?

Reply:
well... the idea was to have the ability to do both... I'll just remove the branch then, to make this more clear.

Reply:
better?